### PR TITLE
fix(all): fix two dates that were wrongly formatted

### LINKED ISSUE #2187: Problem in the format of the `uid` of the Dump response
**Describe the bug**
The `uid` field in response when creating dumps is bad formatted.

**To Reproduce**
Post a request on `/dumps/` route
```
curl \
  -X POST 'http://localhost:7700/dumps'
```
Response:
```
{
	"uid": "%Y%m%d-%H%M%S%3f",
	"status": "in_progress",
	"startedAt": "2022-02-21T18:07:09.436965586Z"
}
```

**Expected behavior**
Response expected:
```
{
	"uid": "20220221-181401337",
	"status": "in_progress",
	"startedAt": "2022-02-21T18:14:01.337987763Z"
}
```

**Meilisearch version:** v0.26.0-rc1

**Additional context**
The access to the dumps status route is now impossible

### LINKED ISSUE #2188: Get stats lastUpdate format issue in v0.26.0rc1
**Describe the bug**
Prior to v0.26.0rc1 the `lastUpdate` for the stats route was formatted as `2022-02-21T22:31:05.182855908Z` however in v0.26.0rc1 I am getting it as `[2022, 52, 22, 34, 26, 300715005, 0, 0, 0]`. Is this change expected?

**To Reproduce**
`curl -X GET 'http://localhost:7700/stats'`

**Expected behavior**
`lastUpdate` formatted as a date

old format

```py
{'databaseSize': 94208, 'lastUpdate': '2022-02-21T22:31:05.182855908Z', 'indexes': {'indexUID': {'numberOfDocuments': 0, 'isIndexing': False, 'fieldDistribution': {}}, 'indexUID2': {'numberOfDocuments': 0, 'isIn
dexing': False, 'fieldDistribution': {}}}}
```

new format

```py
{'databaseSize': 94208, 'lastUpdate': [2022, 52, 22, 34, 26, 300715005, 0, 0, 0], 'indexes': {'indexUID': {'numberOfDocuments': 0, 'isIndexing': False, 'fieldDistribution': {}}, 'indexUID2': {'numberOfDocuments'
: 0, 'isIndexing': False, 'fieldDistribution': {}}}}
```

**Meilisearch version:** v0.26.0rc1

### PR DESCRIPTION
**Pull request by irevoire**
Fix #2188 and fix #2187

### CODE REVIEW

File: meilisearch-lib/src/index_controller/dump_actor/actor.rs [hunk 1]
```diff
@@ -157,3 +159,33 @@ impl DumpActor {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[actix_rt::test]
+    async fn test_generate_uid() {
```
Comment (MarinPostma) [inline meilisearch-lib/src/index_controller/dump_actor/actor.rs:168]:
```suggestion
fn test_generate_uid() {
```

Comment (MarinPostma) [reply meilisearch-lib/src/index_controller/dump_actor/actor.rs:168]: no need for this to be async

**Review by MarinPostma — CHANGES_REQUESTED** [review]

File: meilisearch-lib/src/index_controller/dump_actor/actor.rs (hunk 1 above)
Comment (irevoire) [reply meilisearch-lib/src/index_controller/dump_actor/actor.rs]: That’s true, done 👍

**Review by MarinPostma — APPROVED** [review]

**Comment by irevoire** [comment]
bors merge

**Pull request merged**